accurest-core/src/main/groovy/io/codearte/accurest/util/ContentUtils.groovy

```diff
@@ -57,7 +62,57 @@ class ContentUtils {
 				return extractValueForGString(bodyAsValue, valueProvider)
 			}
 		}
+	}
+
+	public static ContentType getClientContentType(GString bodyAsValue) {
```

Comment:
I think we should assume contentType is a part of contract definition. It should be at least